### ISSUE TITLE
Documentation for Releases & Deployment 

### ISSUE THREAD

**Issue by mihow** [RolnickLab/antenna#648]
Notes for the release process from our call

- Merge PR into main
- Preview environment auto-deploys front & backend
- Regression testing
- - Document core features to test, write a test plan
- - Assign manual testing
- - Write cypress tests & automate
- RELEASE BUTTON (in TeamCity? or script in repo)
- Create a new tag with cal+semver version 
- Create a new release with notes in GitHub (for major releases)
- Choose which environments need to be deployed
- Merge tag into environment branch - (deployments/antenna.insectai.org)
- Or! make PR from main into environment (deployments/production)

Hotfixes?

**Comment by mihow** [comment]
Related to #413

**Comment by mihow** [comment]
@f-PLT @annavik let's discuss these and formalize and add to the Wiki! can you make sense of my existing notes in the description of this ticket?

**Comment by annavik** [comment]
Yes, one thing I could start with here is a simple regression test plan!

For documenting the full release process, I think it might be a bit early still? For me it would be helpful to run through the process at least once before I do that. We haven't been able to that yet (at least not in they way as described in the notes), and I'm not sure if technically possible yet? We would need some scripts and things in place first, right?

We can of course try summarize the notes as a start, but I think what we add to a Wiki should reflect reality, to avoid any confusions for new team members :)

**Comment by mihow** [comment]
Dependent on #413

**Comment by annavik** [comment]
Moving this to sprint 17, I don't think we will unblock this for current sprint.
